guides/use-microprofile-config.adoc

```diff
@@ -0,0 +1,131 @@
+= Use Eclipse MicroProfile Config
+:summary: Learn how to use MicroProfile Config With WildFly.
+:includedir: _includes
+include::{includedir}/_attributes.adoc[]
+:prerequisites-time: 10
+
+In this guide, you will learn how to setup and use Eclipse MicroProfile Config in your application.
+
+include::{includedir}/_prerequisites.adoc[]
+
+[[requirements]]
+== Requirements
+
+In order to use MicroProfile Config in your application, you need to update the Maven `pom.xml` at different places:
+
+1. Add a dependency to WildFly BOM for MicroProfile in the `<dependencyManagement>` section
+2. Add a dependency to the MicroProfile Config API in the `<dependencies>` section
+
+Once these 2 steps will be achieved, you will be able to use MicroProfile Config in your application.
```

Comment:
s/steps will be acheived/steps have been completed,

Reply:
Fixed